ServerRoomMonitoring.Api/Listeners/RabbitMqListener.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using Newtonsoft.Json;
+using RabbitMQ.Client;
+using RabbitMQ.Client.Events;
+using ServerRoomMonitoring.Api.Config;
+using ServerRoomMonitoring.Api.Models;
+using ServerRoomMonitoring.Api.Services;
+
+namespace ServerRoomMonitoring.Api.Listeners
+{
+    public class RabbitMqListener: BackgroundService
+    {
+        private readonly IModel _channel;
+
+        private ISensorService _sensorService;
+        public RabbitMqListener(IRabbitConfig rabbitConfig, ISensorService sensorService)
+        {
+            _sensorService = sensorService;
+            var factory = new ConnectionFactory { HostName = rabbitConfig.HostName };
+            
+            var connection = factory.CreateConnection();
+            connection.ConnectionShutdown += RabbitMQ_ConnectionShutdown;
+            _channel = connection.CreateModel();
+            _channel.QueueDeclare(queue: "server_queue", durable: true, exclusive: false, autoDelete: false, arguments: null);
+        }
+
+        private void RabbitMQ_ConnectionShutdown(object? sender, ShutdownEventArgs e)
+        {
+            Console.WriteLine("RabbitMQ_ConnectionShutdown");
```

Comment:
maybe console logging instead of writing to console. It can give more info like timestamp and class which logged the message.